### PR TITLE
Fix markdown issues

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

**Comment by PRMerger9** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger9** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger15** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger12** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger14** [comment]
@Youssef1313 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Youssef1313** [comment]
There is still more to fix. Marking as ready for review to get an initial review. But it's not ready to merge yet.

**Comment by ktoliver** [comment]
@ghogen and @Mikejo5000 - Could you review this pull request? Thanks.

**Comment by ghogen** [comment]
@Youssef1313 Does this PR enable markdownlint for the repo?  We discussed this internally a while back, and decided not to enable it, to avoid excessive randomization.  I'd suggest holding off on any further work on this, unless that decision is changed.

**Comment by Youssef1313** [comment]
@ghogen, Yes it enables it for the repo.
The workflow running on my fork detected actual problems on the live docs site. So, not all of the changes are just markdown stylish things.

File: docs/deployment/quickstart-deploy-to-azure.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ This article describes how to use the **Publish** tool for one-time deployment.
 
     ![Choose publish target](../deployment/media/quickstart-publish-azure-new.png)
 
-1. Select **Azure App Service (Windows) **and **Next**.
+1. Select **Azure App Service (Windows)** and **Next**.
```
Comment (Youssef1313) [inline docs/deployment/quickstart-deploy-to-azure.md:36]: @ghogen This is one example where it affected how the text is actually rendered.

Comment (ghogen) [reply docs/deployment/quickstart-deploy-to-azure.md:36]: @Youssef1313 OK, thanks. Could you remove the part that enables markdownlint, so we can just accept the other changes?

Comment (Youssef1313) [reply docs/deployment/quickstart-deploy-to-azure.md:36]: @ghogen I have deleted the part that enables the workflow for the repo.

**Pull request merged**